### PR TITLE
fix(language): pt_BR translation

### PR DESCRIPTION
**Pull request by diegolnasc**

### CODE REVIEW

**Review by mistercrunch — APPROVED** [review]

File: superset/translations/pt_BR/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -160,15 +160,15 @@ msgstr "!= (diferente)"
 
 #, fuzzy, python-format
 msgid "% calculation"
-msgstr "Tipo de cálculo"
+msgstr "% calculo"
 
 #, fuzzy, python-format
 msgid "% of parent"
-msgstr "Pai"
+msgstr "% of pai"
```
Comment (michael-s-molina) [inline superset/translations/pt_BR/LC_MESSAGES/messages.po:167]:
```suggestion
msgstr "% do pai"
```

File: superset/translations/pt_BR/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -160,15 +160,15 @@ msgstr "!= (diferente)"
 
 #, fuzzy, python-format
 msgid "% calculation"
-msgstr "Tipo de cálculo"
+msgstr "% calculo"
```
Comment (michael-s-molina) [inline superset/translations/pt_BR/LC_MESSAGES/messages.po:163]:
```suggestion
msgstr "cálculo %"
```

File: superset/translations/pt_BR/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -160,15 +160,15 @@ msgstr "!= (diferente)"
...
 
 #, fuzzy, python-format
 msgid "% of parent"
-msgstr "Pai"
+msgstr "% of pai"
 
 #, fuzzy, python-format
 msgid "% of total"
-msgstr "Mostrar total"
+msgstr "% of total"
```
Comment (michael-s-molina) [inline superset/translations/pt_BR/LC_MESSAGES/messages.po:171]:
```suggestion
msgstr "% total"
```

**Review by michael-s-molina — CHANGES_REQUESTED** [review]

**Review by michael-s-molina — APPROVED** [review]

**Comment by michael-s-molina** [comment]
Obrigado!

**Comment by willianxzy** [comment]
Could anyone help me? I want to translate Apache Superset into Brazilian Portuguese, but I change the messages.po files and compile them and only some things are translated, even trying to change the source code of the react files I can't get any changes in the frontend, could you help me?

print1
print2

**Comment by willianxzy** [comment]
Could anyone help me? I want to translate Apache Superset into Brazilian Portuguese, but I change the messages.po files and compile them and only some things are translated, even trying to change the source code of the react files I can't get any changes in the frontend, could you help me?

print1
print2correto

**Pull request merged**